packages/auth-providers/clerk/web/src/clerk.tsx

```diff
@@ -26,7 +25,9 @@ export function createAuth(customProviderHooks?: {
 function createAuthImplementation() {
   return {
     type: 'clerk',
-    client: (window as any).Clerk as Clerk,
+    get client(): Clerk | undefined {
```

Comment:
Clerk is the only auth provider (I think) that uses a getter method, so maybe we should explain why we're doing that
```suggestion
    // Using a getter here to make sure we're always returning a fresh value
    // and not creating a closure around an old (probably `undefined`) value
    // for Clerk that'll we always return, even when Clerk on the window object
    // eventually refreshes
    get client(): Clerk | undefined {
```